specification/engagementfabric/resource-manager/Microsoft.EngagementFabric/preview/2018-09-01-preview/EngagementFabric.json

```diff
@@ -0,0 +1,1280 @@
+{
+  "swagger": "2.0",
```

Comment:
Please rename directory `2018-09-01-preview` to `2018-09-01`

Reply:
Fixed